metadata-ingestion/source_docs/druid.md

```diff
@@ -0,0 +1,54 @@
+# Druid
+
+## Setup
+
+To install this plugin, run `pip install 'acryl-datahub[druid]'`.
+
+## Capabilities
+
+This plugin extracts the following:
+
+- Metadata for databases, schemas, and tables
+- Column types associated with each table
+
+**Note**: It is important to explicitly define the deny schema pattern for internal Druid databases (lookup & sys) if adding a schema pattern. Otherwise, the crawler may crash before processing relevant databases. This deny pattern is defined by default but is overriden by user-submitted configurations.
+
+## Quickstart recipe
+
+Check out the following recipe to get started with ingestion! See [below](#config-details) for full configuration options.
+
+```yml
+source:
+  type: druid
+  config:
+    host_port: "localhost:8082"
+
+    username: admin
```

Comment:
In the future, we should separate these config sections with shared categorizations in the comments... for example:

```
source:
  type: druid
  config:
    # Coordinates 
    host_port: "localhost:8082"

    # Credentials 
    username: admin
    password: password 

    # Options 
     ....
```